### DISCUSSION TITLE
ESP32-C3 Supermini serial not working and unable to connect to wifi

### DISCUSSION THREAD

**Discussion by ClayF** [ESPresense/ESPresense#996] [Support]
I'm not sure if the board is just unsupported or if this is a bug, but I've tested Tasmota to see if it had the same issues and the web console was working. With ESPresence installed Windows intermittently says the device has malfunctioned. May go away after one or two reboots. When USB does work, the web console say it has connected but there is no output. Putty does not connect at all.

The Change Wi-Fi button also doesn't seem to show up on Web Tools so I'm unable to connect to a hidden network.

Everything else seems to function correctly other that. I got one set up and working pretty well.

**Comment by DTTerastar** [comment]
If the serial isn't working, try the CDC flavor.  CDC is for when the onboard serial is used instead of an external ttl to usb serial chip.

**Reply by ClayF** [reply]
Sorry forgot to.mention, I am using the CDC release
